src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,57 @@
 package core.basesyntax;
 
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.util.Collections;
+import java.util.List;
+
 public class WorkWithFile {
+    private static final String STRING_BUY = "buy";
+    private static final String STRING_SUPPLY = "supply";
+    private static final String STRING_СOMA = ",";
+    private static final int ZERO = 0;
+    private static final int ONE = 1;
 
     public void getStatistic(String fromFileName, String toFileName) {
+        File fileReader = new File(fromFileName);
+        int[] buyAndSupply = new int[2];
```

Comment:
```suggestion
    int totalBuy = 0;
    int totalSupply = 0;
```
let's use 2 variables instead of array